tox.ini

```diff
@@ -126,7 +126,9 @@ basepython = python2.7
 deps =
     -cconstraints.txt
     zest.releaser[recommended]
+    check-manifest
```

Comment:
zest.releaser already uses check-manifest and as long as the @plone/release-team does the releases we are already using our own tooling (in buildout.coredev) to make the releases. Is there any real benefit to keep this here?

Reply:
Ok, I have removed it